compiler/ballerina-backend-jvm-old/src/main/ballerina/src/compiler_backend_jvm/jvm_terminator_gen.bal

```diff
@@ -796,7 +797,7 @@ type TerminatorGenerator object {
 
         } else {
             self.mv.visitMethodInsn(INVOKEVIRTUAL, FUNCTION_POINTER, "call", io:sprintf("(L%s;)L%s;", OBJECT, OBJECT), false);
-            // store result
+            // store reult
```

Comment:
typo ?

Reply:
This was a revert back to original code. I fixed the type but revert goes back to the typo.
And this code will be removed soon from Ballerina. This is an old compiler

Reply:
Revert commit: 0a4b358